### PR TITLE
add com.larksuite.Lark

### PR DESCRIPTION
**Pull request by madoka773**
### Please confirm your submission meets all the criteria

Please describe your application briefly below.

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by madoka773** [comment]
I couldn't find a suitable screenshot of the application on the official website, so I made it myself

File: com.larksuite.Lark.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        only-arches:
+          - x86_64
+        sha256: ff88eb1fdf4ba583a71939ca9fab992306dfa4a796a64d5be81344652d935984
+        filename: lark.deb
+        size: 332307960
+
+      - type: extra-data
+        url: https://sf16-va.larksuitecdn.com/obj/lark-artifact-storage/033c3a0e/Lark-linux_x64-6.9.20.deb
+        only-arches:
+          - aarch64
```
Comment (hfiguiere) [inline com.larksuite.Lark.yaml:66]: x64 for aarch64? I doubt this will work

Comment (madoka773) [reply com.larksuite.Lark.yaml]: oops

**Comment by madoka773** [comment]
bot, build com.larksuite.Lark

File: com.larksuite.Lark.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  # Video Meeting
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --talk-name=org.freedesktop.Notifications
```
Comment (bbhtt) [inline com.larksuite.Lark.yaml:18]:
```suggestion
```

Not needed, handled by portal

File: com.larksuite.Lark.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
+app-id: com.larksuite.Lark
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+separate-locales: false
+command: bytedance-lark-stable
```
Comment (bbhtt) [inline com.larksuite.Lark.yaml:8]: Shouldn't this be vulcan-launcher.sh ?

**Comment by bbhtt** [comment]
> If not, I contacted upstream developers about submitting their software to Flathub. Link:

Can you contact the company behind the software? Email or maybe they have a forum or something?

**Pull request closed without merging**